### ISSUE TITLE
Memory leak

### ISSUE THREAD

**Issue by nachoburr** [WilhelmOks/VoronoiLibSwift#2]
Hi, this is working well but creates zombies.

**Comment by WilhelmOks** [comment]
I'll take a look at it.

**Comment by WilhelmOks** [comment]
I checked it with the profiler and it looks like the VEdge is the problem. But I can't find an obvious reason why.

**Comment by nachoburr** [comment]
Yeah, I wasn’t sure. But it adds up. :( beach line may also be an issue;
did you look for zombies?

**Comment by WilhelmOks** [comment]
I used the Zombie profiler but from what I have read, Zombies are over-released objects, so they wouldn't cause memory leaks.

What I found is, that many objects stay in memory and are never released even if the screen is reset it the test app.
The first object that was directly from the lib was VEdge, so this is a possible candidate for memory leaks but it could also be something else and VEdge is just a symptom instead of the cause.

There is also the chance that the leak is actually in the sample app and not in the library itself.

I'm not very experienced in analyzing that kind of stuff and the fact that I more or less blindly copied the logic from the original C# lib, which has garbage collection, also makes it difficult. It was also more than 3 years ago.
